app/models/user.rb

```diff
@@ -343,6 +343,27 @@ def perform_export_photos!
     update exporting_photos: false
   end
 
+  ######### Data import ##################
+  mount_uploader :import, JsonUploader
+
+  def data_import(importedfile)
+    json_string = importedfile.read()
```

Comment:
Style/MethodCallWithoutArgsParentheses: Do not use parentheses for method calls with no arguments.